connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -188,6 +189,17 @@ public boolean isSinkConnector(String connName) {
         return SinkConnector.class.isAssignableFrom(workerConnector.delegate.getClass());
     }
 
+    public ConfigDef getConnectorConfigDef(String connType) {
+        Class<? extends Connector> connectorClass = getConnectorClass(connType);
+        Connector connector = instantiateConnector(connectorClass);
```

Comment:
Right now we separately check for common connector configs. Do we either a) want to do validation of those configs here or b) somehow require that they are added to `ConfigDefs` returned by the `Connector` implementation? (For example, right now we will not validate that `topics` is defined for a sink connector.)